libs/server/Storage/Session/MainStore/MainStoreOps.cs

```diff
@@ -302,6 +302,72 @@ public unsafe GarnetStatus TTL<TContext, TObjectContext>(ref SpanByte key, Store
             return GarnetStatus.NOTFOUND;
         }
 
+        /// <summary>
+        /// Get the absolute Unix timestamp at which the given key will expire.
+        /// </summary>
+        /// <typeparam name="TContext"></typeparam>
+        /// <typeparam name="TObjectContext"></typeparam>
+        /// <param name="key">The key to get the Unix timestamp.</param>
+        /// <param name="storeType">The store to operate on</param>
+        /// <param name="output">Span to allocate the output of the operation</param>
+        /// <param name="context">Basic Context of the store</param>
+        /// <param name="objectContext">Object Context of the store</param>
+        /// <param name="milliseconds">when true the command to execute is PEXPIRETIME.</param>
+        /// <returns>Returns the absolute Unix timestamp (since January 1, 1970) in seconds or milliseconds at which the given key will expire.</returns>
+        public unsafe GarnetStatus EXPIRETIME<TContext, TObjectContext>(ref SpanByte key, StoreType storeType, ref SpanByteAndMemory output, ref TContext context, ref TObjectContext objectContext, bool milliseconds = false)
+            where TContext : ITsavoriteContext<SpanByte, SpanByte, SpanByte, SpanByteAndMemory, long, MainSessionFunctions, MainStoreFunctions, MainStoreAllocator>
+            where TObjectContext : ITsavoriteContext<byte[], IGarnetObject, ObjectInput, GarnetObjectStoreOutput, long, ObjectSessionFunctions, ObjectStoreFunctions, ObjectStoreAllocator>
+        {
```

Comment:
Why not simply call TTL and convert it to unix-time?

Reply:
That was my first implementation, and then I did this implementation and did a microbenchmark to compare the performance. I found this method is much faster than calling editing TTL and converting to Unix time. I don't have the number handy now but it was like 60% difference. TTL + Convert took about 15 ns and this approach was taking about 9 ns. I know its still few `ns`. @TalZaccai If you are fine with this performance difference then I can change it to TTL + Convert logic. Your call

Reply:
Interesting, I wonder why it would take more time, as this would still do the conversion, only in a different area of the code. 

@badrishc do you have a preference here?

Reply:
Reason for the performance difference is because TTL will write its response in `SpanByteAndMemory` and change its available length. Since UnixTimestamp is always bigger than TTL or PTTL value we are override in the existing space in `SpanByteAndMemory` which makes the code to `Rent` a new memory for the response.

One way I though of overcome this is by resetting the length of `SpanByteAndMemory` to initial after getting the response from TTL/PTTL but not sure how safe it is. I think it would be safe as I would assume those a pinned memory and won't be disposed till we right the response back from the server. If you are fine with doing this then I can use TTL/PTTL itself or if there is a better way, let me know I can implement in that manner.

Reply:
@TalZaccai @badrishc Any update? I can make the suggested changes if you guys let me know what you prefer.

Reply:
Since TTL writes directly to the network, it should be fine to have a separate ExpireTime that writes the time directly to the network buffer as this PR does.